docs/basics/command-line.md

```diff
@@ -5,7 +5,7 @@ description: "A simple walkthrough of how to perform basic IPFS operations using
 
 # Basic CLI Operations
 
-This short guide aims to walk you through the basics of using IPFS with the Kubo CLI.  Kubo is [one of multiple IPFS implementations](.ipfs-implementations.md). It is the oldest IPFS implementation and exposes a CLI (among other things).
+This short guide aims to walk you through the basics of using IPFS with the Kubo CLI.  Kubo is [one of multiple IPFS implementations](ipfs-implementations.md). It is the oldest IPFS implementation and exposes a CLI (among other things).
```

Comment:
```suggestion
This short guide aims to walk you through the basics of using IPFS with the Kubo CLI.  Kubo is [one of multiple IPFS implementations](../ipfs-implementations.md). It is the oldest IPFS implementation and exposes a CLI (among other things).
```

Reply:
I tried this locally - your suggestion of adding "../" resolves to "http://localhost:8080/ipfs-implementations/" which isn't the right page. 

The original commit does properly resolve to "http://localhost:8080/basics/ipfs-implementations/". 

Using a single dot would also work and resolves properly:
[one of multiple IPFS implementations](./ipfs-implementations.md)

Reply:
It should be "../ipfs-implementations" since "ipfs-implementations.md" is located in /basics/ whereas "command-line.md" is in "/basics/go/"

Reply:
"command-line.md" is in "/basics/"

Reply:
Oh, sorry @atomicframeworks, had these changes in mind: https://github.com/ipfs/ipfs-docs/pull/1225, where we are moving "command-line.md" to /basics/go/. Lets update the broken link now and will ensure it is changed in the PR with the move in dir